src/main/resources/com/ncc/aif/aida_ontology.shacl

```diff
@@ -700,23 +833,34 @@ aida:CompoundJustificationShape
    a sh:NodeShape ;
    sh:targetClass aida:CompoundJustification ;
 
-   # must have at least two contained justifications
-   sh:property [
-     sh:path aida:containedJustification ;
-     sh:xone (
-       [sh:class aida:TextJustification]
-       [sh:class aida:ImageJustification]
-       [sh:class aida:AudioJustification]
-       [sh:class aida:KeyFrameVideoJustification]
-       [sh:class aida:ShotVideoJustification] ) ;
-     sh:minCount 1 ] ;
+   # must have at least two contained justifications of specified types
+   sh:property
+     aida:CompoundJustificationMinimum,
+     aida:CompoundJustificationTypes ;
 
    # justification must provide confidence
    sh:property aida:RequiredConfidencePropertyShape ;
 
    # may provide an optional source system
    sh:property aida:SystemPropertyShape .
 
+aida:CompoundJustificationMinimum
+   a sh:PropertyShape ;
+   sh:path aida:containedJustification ;
+   sh:minCount 2 ;
+   sh:message "aida:CompoundJustification requires at least 2 justifications" .
```

Comment:
I would say that the minimum has not been mandated, but should be maintained. Because NIST wants compound justifications for evaluation doesn't mean that AIF should support using compound when a simple justification would work. 

The list of NIST restrictions aren't strictly more restrictive. It is simply the format that NIST needs the rdf in to be able to query. I would argue that NIST is abusing compound justifications. 

We should pose this question to the inter-TA working group, but as it makes sense, I'd like to keep it in for now.

Reply:
I agree that it should be posed to the inter-TA working group, presumably this Friday.

I personally think that if a minimum hasn't been mandated, then we shouldn't add it to the SHACL at this time (even if it makes logical sense), particularly as it causes a cognitive dissonance with a file passing validation of "restricted AIF" but failing validation of non-restricted AIF.  Also, I think it could be frustrating to fail AIF validation due to something that wasn't mandated, and in fact was specifically allowed/used by NIST.

That all said, it _does_ sound like a good restriction ultimately and it will be interesting to hear NIST's explanation of why they use compound justifications this way.  Maybe they plan not to do so moving forward, but it eases things for now.